cmd/jaeger/internal/exporters/storageexporter/README.md

```diff
@@ -15,3 +17,50 @@ extensions:
       memstore:
         max_traces: 100000
 ```
+
+### Retry Configuration
+
+The exporter supports configurable retry behavior for failed exports. By default, retry is enabled with the following settings:
+
+- `enabled`: true
+- `initial_interval`: 5s
+- `randomization_factor`: 0.5
+- `multiplier`: 1.5
+- `max_interval`: 30s
+- `max_elapsed_time`: 5m
+
+#### Explanation of Retry Parameters
```

Comment:
why do we need this? Isn't it already documented in OTEL? I don't want to duplicate the docs, just refer to upstream docs.